README.md

```diff
@@ -44,17 +46,24 @@ the distributed effort. But you can still play, especially if you are patient.
 
 ### Windows
 
-Head to the Github releases page at https://github.com/gcp/leela-zero/releases,
-download the latest release, unzip, and launch autogtp.exe. It will connect to
-the server automatically and do its work in the background, uploading results
+- Easiest way (recommended) : 
+If you want to run the latest stable version ("master" branch), head to the Github releases page at https://github.com/gcp/leela-zero/releases, download the latest release for windows, unzip, and launch autogtp.exe.
+It will connect to the server automatically and do its work in the background, uploading results
 after each game. You can just close the autogtp window to stop it.
 
-### macOS and Linux
+- More complicated way (not recommended, if you know what you're doing) :
+else if you want to manually compile your own leela-zero version, read [the main instructions there](https://github.com/wonderingabout/leela-zero#i-want-to-run-leela-zero-now-main-instructions), then follow [the instructions to compile for windows](https://github.com/wonderingabout/leela-zero#example-of-compiling-and-running---windows) there.
```

Comment:
If it's not recommended, we don't explain it. Again, mentioning this can only add confusion.